Assets/Shopify/Unity/UI/NativePayButtonUI.cs

```diff
@@ -99,10 +111,10 @@ public class NativePayButtonUI : MonoBehaviour {
 #endif
 
 #if UNITY_ANDROID
-        private void GenerateAndroidPayImage() {
+        private void GenerateAndroidPayImage(RectTransform rectTransform) {
```

Comment:
Needed?

Reply:
Nope. I'm going to remove this stuff for now until we get around to it.